core/frontend/package.json

```diff
@@ -62,12 +62,10 @@
     "@itwin/core-quantity": "workspace:*",
     "@itwin/certa": "workspace:*",
     "@itwin/eslint-plugin": "^4.0.0-dev.33",
-    "@itwin/webgl-compatibility": "workspace:*",
```

Comment:
Why is this now only a dev dependency?

Reply:
we made this a direct dep, since its pure / side effect free, a few weeks back.
it was somehow still listed as both direct and dev dep, just removing the unnecessary dev dep

Reply:
Thanks, sorry, I misread which section had changed.